.gitlab-ci.yml

```diff
@@ -143,3 +143,24 @@ publish_public_latest:
     IMG_SOURCES: $BUILD_DOCKER_REGISTRY/$PROJECTNAME:$CI_COMMIT_TAG-amd64,$BUILD_DOCKER_REGISTRY/$PROJECTNAME:$CI_COMMIT_TAG-arm64
     IMG_DESTINATIONS: operator:latest
     IMG_SIGNING: "false"
+
+trigger_internal_image:
+  stage: release
+  rules:
+    - if: $CI_COMMIT_TAG
+      when: manual
+    - when: never
+  needs:
+    - intermediate_latest
+  trigger:
+    project: DataDog/images
+    branch: master
```

Comment:
`main` ?

Reply:
good catch

Reply:
ah no sorry this repo still use "master" as default branch name.